scanner/src/main/kotlin/Scanner.kt

```diff
@@ -43,12 +43,12 @@ import org.ossreviewtoolkit.utils.core.log
 
 const val TOOL_NAME = "scanner"
 
-private fun removeConcludedPackages(packages: Set<Package>, scanner: Scanner): Set<Package> =
-    packages.takeUnless { scanner.scannerConfig.skipConcluded }
+private fun removeConcludedPackages(packages: Set<Package>, scanner: Scanner?): Set<Package> =
+    packages.takeUnless { scanner != null && scanner.scannerConfig.skipConcluded }
```

Comment:
This means that now packages are always filtered if the package scanner is null, this does not really make sense, as they are not scanned anyway. I think I'd rather not call this function at all if the scanner is null.